presto-main/src/main/java/com/facebook/presto/execution/SqlTaskExecutionFactory.java

```diff
@@ -26,6 +26,7 @@
 import com.facebook.presto.sql.planner.LocalExecutionPlanner.LocalExecutionPlan;
 import com.facebook.presto.sql.planner.PlanFragment;
 import com.facebook.presto.sql.planner.plan.PlanNodeId;
+import com.facebook.presto.util.GcMonitor;
```

Comment:
I don't see `GcMonitor` in this PR.

Reply:
The final version is in airlift (next to the pause meter). https://github.com/airlift/airlift/pull/607